scipy/stats/tests/test_stats.py

```diff
@@ -7663,6 +7663,20 @@ def test_error_code(self):
             v_values = rng.random(size=(2, 2))
             _ = stats.wasserstein_distance(u_values, v_values)
 
+    def test_flatten(self):
+        rng = np.random.default_rng(456347456756786)
+        num_tests = 100
+        u_values_len_array = rng.integers(low=1, high=10, size=num_tests)
+        v_values_len_array = rng.integers(low=1, high=10, size=num_tests)
+        for i in range(num_tests):
+            u_values = rng.random(size=(u_values_len_array[i],1))
+            v_values = rng.random(size=(v_values_len_array[i],1))
+            u_values_flat = u_values.flatten()
```

Comment:
Doesn't really matter here, but note that `flatten` copies the data and `ravel` reshapes but avoids copying.

Reply:
Thanks.